tools/meteor-services/auth.js

```diff
@@ -65,6 +65,9 @@ var loggedInAccountsConnection = function (token) {
     connection.close();
 
     if (err.error === 403) {
+      // Better than saying nothing at all
+      Console.error("warning: login token was rejected by the server");
```

Comment:
Ok, thanks. I'll take a look at that other location. If I make changes, should I just close out this PR and resubmit as a new one?

The solution was to simply regenerate the token - it was a ~30 sec fix, but it took us quite an effort to figure it out on the CI system since there was no indication of the actual problem.

RE: messaging - agreed, we should take the word warning out of the error message. I think it's still good to have an error so it is more likely to be reported, rather than ```console.warn``` which might be overlooked.